docs/content.zh/docs/dev/datastream/fault-tolerance/serialization/custom_serialization.md

```diff
@@ -24,26 +24,22 @@ specific language governing permissions and limitations
 under the License.
 -->
 
-# Custom Serialization for Managed State
+# 托管状态的自定义序列化
 
 
-This page is targeted as a guideline for users who require the use of custom serialization for their state, covering
-how to provide a custom state serializer as well as guidelines and best practices for implementing serializers that allow
-state schema evolution.
+这个页面是为需要为其状态使用自定义序列化的用户提供的指南，包括如何提供自定义状态序列化器以及实现允许状态模式演进的序列化器的指南和最佳实践。
 
-If you're simply using Flink's own serializers, this page is irrelevant and can be ignored.
+如果你仅使用 Flink 自己的序列化器，那么这个页面对你来说无关紧要，可以忽略。
+## 使用自定义状态序列化器
 
-## Using custom state serializers
```

Comment:
Do need a blank line?

Reply:
Thank you for your suggestion, there is indeed a blank line in front.